x/profiles/types/models_chain_links.go

```diff
@@ -202,6 +204,57 @@ func (b Base58Address) VerifyPubKey(key cryptotypes.PubKey) (bool, error) {
 
 // --------------------------------------------------------------------------------------------------------------------
 
+var _ AddressData = &HexAddress{}
+
+// NewHexAddress returns a new HexAddress instance
+// NOTE: Currently it only supports keccak256-uncompressed addresses
```

Comment:
I would move this note directly inside the Protobuf `message` description, so that it's visible to anyone using this type
```suggestion
```